### DISCUSSION TITLE
Add "duplicate" option to Custom Entities

### DISCUSSION THREAD

**Discussion by proddy** [emsesp/EMS-ESP32#2266] [Ideas]
As requested by JHa on [Discord](https://discord.com/channels/816637840644505620/816958087160266775/1312370816415043696) 

> I also would like to suggest about an enhancement request: Wouldn't it be helpful to have something like "duplicate" on existing entities. Often you just want to change the offset of an already existing entry and a duplicate would help to enter nearly same entities more easy. Just as an new idea

**Comment by proddy** [comment]
We'll see how much effort this is. One quick way is to export the Custom Entities and edit the JSON file using a text editor and re-importing it back into EMS-ESP. It's always good to keep a backup of the files.

**Comment by gh47110815** [comment]
Hey what a surprise ;-)  - I just checked for updates, saw #2266 in the release notes, installed 3.7.2-dev.4 and ..... I was deeply impressed.
Duplicate entry works - very cool - THANK YOU very very much.

**Reply by proddy** [reply]
👍🏻 That was all Michael's great work
